recipes/libtiledb-sql-py/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "libtiledb-sql-py" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "c0228d64bdf9eeabdac5284c760bf4899892a170f769688a09285bcbd46963d1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/TileDB-Inc/TileDB-SQL-Py.git
```

Comment:
Please build from a source tarball: https://conda-forge.org/docs/maintainer/adding_pkgs.html#tarballs-no-repos

Reply:
I've reworked the package to use `setuptools_scm_git_archive`, it no longer needs the git repo and now uses a github archive.